skel/problem/generators/generators.yaml

```diff
@@ -3,7 +3,7 @@
 
 data:
   sample:
-    data:
+    data: []
```

Comment:
Good idea to make the skeleton spec-compliant like this! Do you think we should explain that a problem setter should remove the `[]` to make use of the commented-out examples below this? I guess this could lead to confusion, especially to people who are new to BAPCtools :slightly_smiling_face:

Reply:
I have no opinion either way; and just did the minimally intrusive edit. Others should retain the editorial responsibility for the skeleton. (Which contains _a lot_ of information, for instance about repurposing symlinked input validators as generators, which I believe is at least as confusing. I think it’s good enough as it is, and the didactics are a different issue in any case.)